Demo/Sources/SettingsView.swift

```diff
@@ -87,6 +88,19 @@ struct SettingsView: View {
         }
     }
 
+    @ViewBuilder
+    private func versionSection() -> some View {
+        Section {
+            EmptyView()
+        } footer: {
+            HStack {
+                Text("Version \(Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? "")")
+                Text("Build \(Bundle.main.infoDictionary?["CFBundleVersion"] as? String ?? "")")
```

Comment:
Improvement proposals:

1. These values should be available from the Info.plist so force-unwrapping should be safe and crash if we failed to set these keys.
2. Extracting as private vars could make it even more readable.

So for example:

```swift
private var version: String {
    Bundle.main.infoDictionary!["CFBundleShortVersionString"] as! String
}
```

and similarly for the build number.